### ISSUE TITLE
after created a new admin user got Error 419 Page expired

### ISSUE THREAD

**Issue by Luazzo** [thedevdojo/voyager#5063]
## Version information

 - Laravel: v6.2
 - Voyager: v1.14
 - PHP: 7.3.12
 - Database:  MySQL 8.0.18

## Description

I installed in my project Voyager with dummy and it worked fine,
later i created a new admin user by commande php artisan voyager:admin xxx@email.com --create
it worked the first day but the next day i've got Error 419 | Page expired.

How can I fix this error ?

thanx for help

**Comment by MrCrayon** [comment]
419 means usually you have a problem with sessions, if refreshing doesn't work check that your session folder is writable.
Anyway it's not related to Voyager which BTW never had a v1.14, maybe v1.4.1?

**Comment by Luazzo** [comment]
thanks for your reply!
yes I was wrong with the version of Voyager, it's 1.4.2
I created another new admin and it still works, then the session folder is working?..
but I noticed that the **remember_token** isn't generated in the database
maybe there is a reason?  although hardly...

one more addition:
the admin user that i've created is at localhost
I also have this project on the host server.
And there is only one admin user created by Voyager (admin @ admin / password), but I get the same 419 error...
